### PR TITLE
[front] add missing breadcrumbs in customization+ fix(#8274)

### LINKED ISSUE #8274: Missing breadcrumb in multiple entities/objects overview
## Description

Missing breadcrumb in user / group / role overviews

### PR DESCRIPTION
**Pull request by CelineSebe**
### Proposed changes

* breadcrumbs in customization

### Related issues

* #8274 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/settings/decay/DecayRule.tsx [hunk 1]
```diff
@@ -108,6 +109,11 @@ const DecayRuleComponent = ({ queryRef }: DecayRuleComponentProps) => {
 
   return (
     <div className={classes.container}>
+      <Breadcrumbs variant="list" elements={[{ label: t_i18n('Settings') }, { label: t_i18n('Customization') }, {
+        label: t_i18n('Decay rules'),
+        link: '/dashboard/settings/customization/decay',
+      }, { label: decayRule.name, current: true }]}
+      />
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/settings/decay/DecayRule.tsx:118]: bad indentation

File: opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx [hunk 2]
```diff
@@ -16,13 +18,22 @@ export const subTypeQuery = graphql`
...
   const data = useLazyLoadQuery<RootSubTypeQuery>(subTypeQuery, { id: subTypeId });
 
   return (
     <Suspense fallback={<Loader />}>
       {
-        data.subType ? <SubType data={data.subType} /> : <ErrorNotFound />
+        data.subType ? (
+          <>
+            <Breadcrumbs variant="list" elements={[{ label: t_i18n('Settings') }, { label: t_i18n('Customization') }, {
+              label: t_i18n('Entity types'),
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx]: indentation

File: opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx [hunk 3]
```diff
@@ -16,13 +18,23 @@ export const subTypeQuery = graphql`
...
     <Suspense fallback={<Loader />}>
       {
-        data.subType ? <SubType data={data.subType} /> : <ErrorNotFound />
+        data.subType ? (
+          <>
+            <Breadcrumbs variant="list" elements={[
+              { label: t_i18n('Settings') },
+              { label: t_i18n('Customization') },
+              { label: t_i18n('Entity types'), link: '/dashboard/settings/customization/entity_types' },
+            ]}
```
Comment (marieflorescontact) [inline opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx:33]: indentation

**Review by marieflorescontact — APPROVED** [review]
except indentation it looks good

File: opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx (hunk 2 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx]: thanks !

**Pull request merged**